packages/twenty-front/src/modules/object-record/object-filter-dropdown/components/ObjectFilterDropdownFilterSelect.tsx

```diff
@@ -64,18 +76,14 @@ export const ObjectFilterDropdownFilterSelect = () => {
           setSearchText(event.target.value)
         }
       />
+      <SelectableList hotkeyScope={FiltersHotkeyScope.ObjectFilterDropdownButton}  selectableItemIdArray={refactoredAvailableFilterDefinitions.map((item)=>item.fieldMetadataId)} selectableListId={OBJECT_FILTER_DROPDOWN_ID}>
       <DropdownMenuItemsContainer>
-        {[...availableFilterDefinitions]
-          .sort((a, b) => a.label.localeCompare(b.label))
-          .filter((item) =>
-            item.label
-              .toLocaleLowerCase()
-              .includes(searchText.toLocaleLowerCase()),
-          )
+        {refactoredAvailableFilterDefinitions
           .map((availableFilterDefinition, index) => (
             <MenuItem
```

Comment:
Can't you use `MenuItemSelect` instead ? Seems like it's a better match for this use case.